scripts/tests/scripts_regression_tests.py

```diff
@@ -1399,6 +1403,29 @@ def test_cime_case(self):
             self.assertEqual(case.get_value("TESTCASE"), "TESTRUNPASS")
 
     ###########################################################################
+    def test_cime_case_prereq(self):
```

Comment:
You could include a negative test - make the first run fail and assure that the second doesn't start.    As for the jobid, code env_batch.py has a subroutine get_job_id.   Since you will need to submit two separate jobs for this test you should look at the ERR test for an example.